shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/metadata/generator/PipelineDDLGenerator.java

```diff
@@ -46,8 +56,35 @@
 @RequiredArgsConstructor
 public final class PipelineDDLGenerator {
     
+    private static final String DELIMITER = ";";
+    
+    private static final String NEWLINE = "\n";
+    
     private final ContextManager contextManager;
     
+    /**
+     * Generate logic ddl sql.
+     *
+     * @param databaseType database type
+     * @param databaseName database name
+     * @param schemaName schema name
+     * @param tableName table name
+     * @return ddl SQL
+     */
+    @SneakyThrows
+    public String generateLogicDDLSQL(final DatabaseType databaseType, final String databaseName, final String schemaName, final String tableName) {
+        ShardingSphereMetaData metaData = contextManager.getMetaDataContexts().getMetaData(databaseName);
+        String sql = generateActualDDLSQL(databaseType, schemaName, tableName, metaData);
+        String[] multiSql = sql.split(DELIMITER);
```

Comment:
Please refactor this param to inline expression.

Reply:
Thank you. Done.